components/mysql/sources/new-table/new-table.js

```diff
@@ -0,0 +1,26 @@
+const common = require("../common.js");
+
+module.exports = {
+  ...common,
+  key: "mysql-new-table",
+  name: "New Table",
+  description: "Emits an event when a new table is added to a database",
+  version: "0.0.1",
+  dedupe: "unique",
+  methods: {
+    ...common.methods,
+    async listResults(connection) {
+      const tables = await this.mysql.listTables(connection);
+      this.iterateAndEmitEvents(tables);
```

Comment:
Probably a good use case for a query like this:

```sql
SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'some_table';
```

This will retrieve more useful information, including `CREATE_TIME` which you can use to detect if a table is new or not.

Reply:
Updated